### PR TITLE
Update Cluster management CLI (add, join, and more)

### PR DESCRIPTION
**Pull request by louiseschmidtgen**
## Description
This PR aims to address changes made to the Cluster management CLI. 
Note: We will not do any API changes yet as they are under discussion.
Overview of what has changed:
#### k8s get-join-token
- `k8s add-node` >> `k8s get-join-token`
- rename `k8s_add_node.go` >> `k8s_get_join_token.go `
- Not implemented:`POST /k8sd/cluster/tokens` >> `POST /k8sd/cluster/get-join-token`
#### k8s join-cluster
- `k8s join-cluster <token>` >> `k8s join-cluster <join_token>`
- `join-node` >> `join-cluster`
- Not implemented:`POST /1.0/k8sd/cluster/join` >> `POST /1.0/k8sd/cluster/nodes`
#### k8s status
- status output: `"k8s is ready."` >> `status: ready`

#### k8s remove-node
- `remove-node <name>` >> `remove-node <node_name>`
-  Not implemented: `DELETE /k8sd/cluster/remove` >> `DELETE /k8sd/cluster/nodes/{id}`
## Testing
- run all changed commands

### CODE REVIEW

File: src/k8s/cmd/k8s/k8s_join_cluster.go [hunk 1]
```diff
@@ -15,48 +15,48 @@ import (
...
+			"Make sure that the name provided in `k8s get-join-token` matches the hostname of the " +
 			"joining node or asign another name with the `--name` flag",
 	}
 )
 
-func newJoinNodeCmd() *cobra.Command {
+func newJoinClusterCmd() *cobra.Command {
 	joinNodeCmd := &cobra.Command{
-		Use:     "join-cluster <token>",
+		Use:     "join-cluster <join_token>",
```
Comment (neoaggelos) [inline src/k8s/cmd/k8s/k8s_join_cluster.go:34]:
```suggestion
		Use:     "join-cluster <join-token>",
```

File: src/k8s/cmd/k8s/k8s_get_join_token.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		worker bool
+	}
+	getJoinTokenCmdErrorMsgs = map[error]string{
+		apiv1.ErrTokenAlreadyCreated: "A token for this node was already created and the node did not join.",
+	}
+)
+
+func newGetJoinTokenCmd() *cobra.Command {
+	getJoinTokenCmd := &cobra.Command{
+		Use:     "get-join-token <node_name>",
```
Comment (neoaggelos) [inline src/k8s/cmd/k8s/k8s_get_join_token.go:22]:
```suggestion
		Use:     "get-join-token <node-name>",
```

File: src/k8s/cmd/k8s/k8s_remove_node.go [hunk 3]
```diff
@@ -18,7 +18,7 @@ var (
 
 func newRemoveNodeCmd() *cobra.Command {
 	removeNodeCmd := &cobra.Command{
-		Use:     "remove-node <name>",
+		Use:     "remove-node <node_name>",
```
Comment (neoaggelos) [inline src/k8s/cmd/k8s/k8s_remove_node.go:21]:
```suggestion
		Use:     "remove-node <node-name>",
```

File: docs/src/tutorial/add-remove-nodes.md [hunk 4]
```diff
@@ -71,7 +71,7 @@ A base64 token will be printed to your terminal. Keep it handy as you will need
 To join the worker node to the cluster, run:
 
 ```sh
-sudo k8s join-cluster <token>
+sudo k8s join-cluster <join_token>
```
Comment (neoaggelos) [inline docs/src/tutorial/add-remove-nodes.md:74]:
```suggestion
sudo k8s join-cluster <join-token>
```

**Review by neoaggelos — COMMENTED** [review]
Looks good, nice work! Let's not mix `_` and `-`

File: src/k8s/cmd/k8s/k8s_join_cluster.go [hunk 5]
```diff
@@ -15,48 +15,48 @@ import (
...
 	}
-	joinNodeCmdErrorMsgs = map[error]string{
+	joinClusterCmdErrorMsgs = map[error]string{
 		apiv1.ErrAlreadyBootstrapped: "A bootstrap node cannot join a cluster as it is already in a cluster. " +
 			"Consider reinstalling the k8s snap and then join it.",
-		apiv1.ErrInvalidJoinToken: "The provided token is not valid. " +
-			"Make sure that the name provided in `k8s add-node` matches the hostname of the " +
+		apiv1.ErrInvalidJoinToken: "The provided join token is not valid. " +
+			"Make sure that the name provided in `k8s get-join-token` matches the hostname of the " +
 			"joining node or asign another name with the `--name` flag",
```
Comment (bschimke95) [inline src/k8s/cmd/k8s/k8s_join_cluster.go:28]: nit: assign

File: src/k8s/pkg/k8sd/api/cluster_remove.go [hunk 6]
```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/canonical/microcluster/state"
 )
 
-func postClusterRemove(m *microcluster.MicroCluster, s *state.State, r *http.Request) response.Response {
+func postClusterRemoveNode(m *microcluster.MicroCluster, s *state.State, r *http.Request) response.Response {
```
Comment (bschimke95) [inline src/k8s/pkg/k8sd/api/cluster_remove.go:17]: the name of those functions should match the API endpoint.
```
POST /cluster/remove
```
--> `postClusterRemove`

**Review by bschimke95 — COMMENTED** [review]
great work, thanks for the clean-up.
only two small nits then we are ready to merge.

**Review by bschimke95 — APPROVED** [review]
LGTM

**Review by neoaggelos — COMMENTED** [review]
Please rebase

**Review by neoaggelos — APPROVED** [review]

**Pull request merged**